apps/meteor/app/lib/server/functions/relinquishRoomOwnerships.ts

```
@@ -1,30 +1,65 @@
+			if (!team) {
+				throw new Error('error-team-not-found');
+			}
+
+			if (!userId) {
+				throw new Error('error-user-not-found');
+			}
+
+			await eraseTeam(userId, team, []);
+
+			resolve();
+		});
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)